python/doc/pyplots/Kent.py

```
@@ -0,0 +1,117 @@
+kappa_vals = [5.0, 20.0]
+beta_vals = [1.0, None]
+
+fig = plt.figure(figsize=(16, 8))
+index = 0
+factor = 0.3
+
+for kappa in kappa_vals:
+    for beta in enumerate(beta_vals):
+        if beta is None:
+            beta = 0.45 * kappa
+        for g2, g3 in zip(gamma2_list, gamma3_list):
```

Ruff (0.15.15)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)